src/modules/video_player/index.js

```diff
@@ -57,7 +57,31 @@ function handlePlayerClick() {
 }
 
 function handlePlayerScroll(event) {
+  maybeSeek(event);
+  maybeControlVolume(event);
+}
+
+function maybeSeek (event) {
```

Comment:
these changes need to be formatted with prettier

Reply:
![image](https://user-images.githubusercontent.com/6674662/196748973-e01c21c7-f00c-4216-a3d9-42c6b5508b77.png)
I'm getting a ton of linting errors when running prettier. Looks like maybe something with line endings on my end? I'm running on Windows.

Reply:
Read into this a bit more. Looks like all files are formatted with unix line endings and prettier is configured to enforce that. However,  running `npm run eslint --fix` immediately converts it back to dos line endings for me. Does it make sense to keep this rule or should we disable it?